src/client/src/commons/form/profile/components/profileLayers/components/lithologicalDescriptionLayers/actionButtons.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Stack, Tooltip } from "@mui/material";
+import { useTranslation } from "react-i18next";
+import ClearIcon from "@mui/icons-material/Clear";
+import ModeEditIcon from "@mui/icons-material/ModeEdit";
+import DeleteIcon from "@mui/icons-material/Delete";
+import AddCircleIcon from "@mui/icons-material/AddCircle";
+
+const ActionButtons = props => {
+  const {
+    item,
+    changeSelectionAndSubmit,
+    addMutation,
+    selectedDescription,
+    selectedStratigraphyID,
+    setDescriptionIdSelectedForDelete,
+  } = props;
```

Comment:
Q: Anstatt die props wie hier auszupacken kann man sie auch direkt in der definition entpacken. Würde das hier sinn machen?
```
const ActionButton = ({item, changeSelectionAndSubmit, ... }) => { ...
```

Reply:
Können wir in Zukunft auch so machen, wir haben das bisher noch nirgends in der Applikation drin. Ich finde es hat weder offensichtliche Vorteile, noch Nachteile, oder übersehe ich etwas?